mmdet/models/backbones/csp_darknet.py

```diff
@@ -258,23 +263,23 @@ def __init__(self,
             self.add_module(f'stage{i + 1}', nn.Sequential(*stage))
             self.layers.append(f'stage{i + 1}')
 
-    def _freeze_stages(self):
+    def _freeze_stages(self) -> None:
         if self.frozen_stages >= 0:
             for i in range(self.frozen_stages + 1):
                 m = getattr(self, self.layers[i])
                 m.eval()
                 for param in m.parameters():
                     param.requires_grad = False
 
-    def train(self, mode=True):
+    def train(self, mode: bool = True) -> None:
         super(CSPDarknet, self).train(mode)
```

Comment:
check whether can use super().train(mode)

Reply:
no need to change